cilium-operator-generic/Dockerfile

```diff
@@ -6,16 +6,14 @@ ARG UBUNTU_IMAGE=ghcr.io/cybozu/ubuntu:22.04
 FROM ${GOLANG_IMAGE} as build
 
 COPY TAG /
-COPY fix-metallb-bug.patch /tmp/
```

Comment:
https://github.com/cilium/cilium/commit/8298b0fd5c76bc5d3ea2778b85317d247e59d946 has been merged since cilium 1.14, so the patch seems to be unnecessary now.

Reply:
Thank you!